NineChronicles.Headless.Executable/Program.cs

```diff
@@ -190,6 +189,12 @@ static void ConfigureSentryOptions(SentryOptions o)
                 "Determines the type of transport.  \"netmq\" and \"tcp\" " +
                 "is available and \"tcp\" option is selected by default.")]
             string transportType = "tcp",
+            [Option(Description = "Node ID used for PBFT consensus.  0 by default.")]
+            long nodeId = 0,
+            [Option(Description =
+                "List of validator's public key strings. " +
+                "If null is given, value in policy will be used.  null by default.")]
+            string[]? validatorStrings = null,
```

Comment:
I suggest using [`IBlockPolicy.GetValidators`](https://github.com/planetarium/libplanet/blob/pbft/Libplanet/Blockchain/Policies/IBlockPolicy.cs#L146-L151) instead argument.

Reply:
`nodeId` is equal to the miner's public key index on the `GetValidators` list.